content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -260,7 +260,7 @@ Please select one of the tabs to see installation instructions for the respectiv
 {{% tab name="Calico" %}}
 For more information about using Calico, see [Quickstart for Calico on Kubernetes](https://docs.projectcalico.org/latest/getting-started/kubernetes/), [Installing Calico for policy and networking](https://docs.projectcalico.org/latest/getting-started/kubernetes/installation/calico), and other related resources.
 
-In order for Network Policy to work correctly, you need to pass `--pod-network-cidr=192.168.0.0/16` to `kubeadm init`. Note that Calico works on `amd64` only.
+For Calico to work correctly, you need to pass `--pod-network-cidr=192.168.0.0/16` to `kubeadm init` or update `calico.yml` file to match your Pod network. Note that Calico works on `amd64` only.
```

Comment:
> or update `calico.yml` file

or update the `calico.yaml` file.
add `the`.

Reply:
We should also probably bump the calico version to v3.2 and then we can remove the `amd64` limitation, since Calico v3.2 also supports arm64 and ppc64le per: https://docs.projectcalico.org/v3.2/releases/

Reply:
@neolit123 thanks, fixed. 

@detiber I'll verify 3.2 with conformance tests and send new PR.